src/test/java/com/openlattice/hazelcast/serializers/EntitySetContainsFlagEntryProcessorStreamSerializerTest.kt

```diff
@@ -0,0 +1,15 @@
+package com.openlattice.hazelcast.serializers
+
+import com.kryptnostic.rhizome.hazelcast.serializers.AbstractStreamSerializerTest
+import com.openlattice.assembler.processors.EntitySetContainsFlagEntryProcessor
+import com.openlattice.mapstores.TestDataFactory
+
+class EntitySetContainsFlagEntryProcessorStreamSerializerTest : AbstractStreamSerializerTest<EntitySetContainsFlagEntryProcessorStreamSerializer, EntitySetContainsFlagEntryProcessor>() {
```

Comment:
Can just make your StreamSerializer `TestableSelfRegisteringStreamSerializer` and just get rid of this :D